rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/client/AuthRestClient.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.sample.superheroes.fight.client;
+
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.core.json.JsonObject;
+
+import jakarta.ws.rs.BeanParam;
+import jakarta.ws.rs.FormParam;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.Response;
+
+import jakarta.ws.rs.core.SecurityContext;
+
+import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import org.jboss.resteasy.reactive.RestPath;
+
+@RegisterRestClient(configKey = "auth-client")
+@RegisterClientHeaders
+@Path("/")
+public interface AuthRestClient {
+  @GET
+  @Path("/q/webauthn/webauthn.js")
+  public Uni<Response> javascript();
+  @POST
+  @Path("/q/webauthn/register")
+  public Uni<Response> setUpObtainRegistrationChallenge(JsonObject jsonObject);
```

Comment:
Since this is one of the default endpoints offered by the `WebAuthn` extension, I went here: https://github.com/ingmarfjolla/quarkus/blob/main/extensions/security-webauthn/runtime/src/main/java/io/quarkus/security/webauthn/WebAuthnController.java#L201 and saw that the routing context was taken as a `JsonObject` and figured it might be the best way to handle that (since I also don't need to access the object as a POJO at any time)